### PR TITLE
Added/Revised T1 Entry EWAR Frigates

### PR DESCRIPTION
**Pull request by Epicloa-Program**
Added detailed descriptions with all the relevant information for each ship. Gives newer players all the information they need to fly the ship effectively.

### CODE REVIEW

File: Fits/Frigates/[ADV-5] Malediction.md
Comment (bearthatcares) [inline Fits/Frigates/[ADV-5] Malediction.md:1]: This shouldn't be apart of this PR

File: Fits/Interceptors/[ADV-5] Ares.md
Comment (bearthatcares) [inline Fits/Interceptors/[ADV-5] Ares.md:1]: Separate branch!!!!

**Review by bearthatcares — CHANGES_REQUESTED** [review]
We already have fittings for all these, example https://tools.minmatar.org/fleets/fittings/nvy-30-vigil/

How do these differ and were we aware of those?

File: Fits/Frigates/[ADV-5] Malediction.md
Comment (Epicloa-Program) [reply Fits/Frigates/[ADV-5] Malediction.md:1]: Sorry I thought I did these as a separate thing, I just needed the file structure from the previous PR so I thought I clicked the right thing.

**Comment by Epicloa-Program** [comment]
> We already have fittings for all these, example https://tools.minmatar.org/fleets/fittings/nvy-30-vigil/
> 
> How do these differ and were we aware of those?

Yeah so the primary differences that I aimed for were:

A) Lowered price, the new ones are roughly half the price of the old (albeit cheap to cheaper but they didn't really lose anything in the process).
B) Cap stability and maximum effectiveness with low SP (used max alpha skills as default for these). Example being the current Vigil fit is 2.9mil and not stable even with all 5s, cap lasts a whopping 60s with max alpha skills. New Vigil is 1.16mil, cap stable with max alpha skills, and has lock-range to actually be at 100km. A consistent issue with the current fits is that they have no caplife and no lock range, especially with lower SP.

**Review by bearthatcares — APPROVED** [review]

**Comment by Epicloa-Program** [comment]
Fixing PRs to avoid conflicts.

**Pull request closed without merging**